docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,715 @@
+Our observations of the particle's position are subject to noise.
+In this section, we model this as Gaussian additive noise on the positions in the track.
+
+Note that free diffusion assumes that the space available for diffusion :math:`L` is sufficiently large that the measurement time interval is much smaller than :math:`L / 2D` :cite:`qian1991single`:
+
+To simulate diffusive tracks with observation noise, we can use :func:`~lumicks.pylake.simulation.simulate_diffusive_tracks`::
+
+    tracks = lk.simulation.simulate_diffusive_tracks(diffusion_constant=1.0, steps=40, dt=0.01, observation_noise=0.1, num_tracks=25)
+    tracks.plot()
+
+.. image:: figures/simulated_tracks.png
+
+We will use this functionality to demonstrate the various methods for estimating diffusion constants implemented in Pylake.
+
+Method background
+-----------------
+
+Consider a freely diffusive track, how would one quantify its diffusivity?
+One convenient way would be to consider its positional increments over time.
+As shown in the simulation model, the increments in free diffusion are independent of the current position and depend directly on the diffusion constant.
+Considering that diffusion is a stochastic process, it is important to average over many such increments to obtain a result not subject to a lot of stochastic error.
+The exact details of how to obtain a diffusion constant depend on the method in question and will be discussed below.
+
+Mean Squared Displacements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Diffusive particles have an average displacement of zero.
+Rather than calculating an average displacement, it makes more sense to consider a form of distance travelled.
+In practice, the mean squared displacement is often used.
+When dealing with pure diffusive motion (a complete absence of drift) in an isotropic medium, the mean squared displacements follow the following relation:
+
+.. math::
+
+    \rho[n] = \langle\left(x_{i} - x_{i + n}\right)^2\rangle = 2 D n \Delta t
+
+Here the brackets :math:`\langle\ldots\rangle` indicate the expected value, :math:`\rho[n]` corresponds to the mean squared displacement for lag :math:`n`, :math:`\Delta t` is the time step of one lag and :math:`D` is the diffusion constant we seek.
+As you can see, the Mean Squared Displacement (MSD) gives a simple relation between the diffusion constant and the observed displacements, which is why MSD analysis is often used for diffusion analysis.
+
+Time-Averaged Mean Squared Displacements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are multiple ways to estimate this quantity in practice.
+We can average over different tracks (ensemble averaging) or over time (time averaging).
+If all tracks experience the same diffusion, the ensemble averaged motion of a collection of independent molecules results in the same diffusion estimates as the time-averaged motion; the diffusive process is 'ergodic'.
```

Comment:
I've dropped the first sentence. The second sentence is kind of important though, since it provides the justification for time averaging the MSDs. Maybe I should make that more explicit. How about:
```
If all tracks experience the same diffusion, the ensemble averaged motion of a collection of independent molecules results in the same diffusion estimates as the time-averaged motion; the diffusive process is 'ergodic'.
As a consequence, we can use time-averaged estimators to estimate diffusion constants on a per-track basis.
```